portfolio/src/main/java/com/google/sps/input/Comments.java

```diff
@@ -14,15 +14,15 @@
 
 package com.google.sps.input;
 
-/** Will be used in the next PR (just setting it up!) */
+// Class that stores comment details
```

Comment:
nit: use java style comments for Class. Sorry I didn't notice this earlier.
/** comments */

Reply:
Done